Library/Homebrew/test/cask/cask_loader/from_uri_loader_spec.rb

```diff
@@ -10,6 +10,11 @@
       expect(described_class.try_new("https://brew.sh/")).not_to be_nil
     end
 
+    it "returns nil when path loading is disabled" do
+      ENV["HOMEBREW_FORBID_PACKAGES_FROM_PATHS"] = "1"
+      expect(described_class.try_new(URI("https://brew.sh/"))).to be_nil
```

Comment:
If this also encompasses URLs, it should have a different name. `HOMEBREW_FORBID_EXTERNAL_PACKAGE_SOURCES`?

Reply:
We do not support non-file URLs anymore: https://github.com/Homebrew/brew/commit/8b0a4a98bf10b3441008906632648977b4a4f23e#diff-aea1b1bc211f85168f88057b630dac95a4947b4abfa99c158ac592090fb3b23d

I'll update this test to clarify this.